.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @dudoslav @ihnorton @jdblischak @johnkerl @nguyenv @shelnutt2 @kounelisagis
\ No newline at end of file
+* @dudoslav @ihnorton @jdblischak @johnkerl @kounelisagis @nguyenv @shelnutt2
```

Comment:
These "errors" will go away once they accept the email invitation from GitHub to join this repository